### DISCUSSION TITLE
Undo Valid Dead move

### DISCUSSION THREAD

**Discussion by tlchim** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#53] [Q&A] [answered]
Valid.Dead is not undoable, should I increase the number of undoes, if the player attempt to undo a Valid.Dead move? Or the step before Valid.Dead will be undo? As this will change the score.

**Comment by Derppening** [comment, marked as answer]
Short Answer: Try to run the obfuscated JAR and see what the behavior is.

Long Answer: Since `Valid.Dead` moves are not undoable, when a player tries to undo a step after making a `Valid.Dead` move, you should undo the latest `Valid.Alive` move.
